WooCommerce/UITestsFoundation/Screens/Login/TwoFAScreen.swift

```diff
@@ -18,10 +22,13 @@ public final class TwoFAScreen: ScreenObject {
         try super.init(
             expectedElementGetters: [
                 twoFAFieldGetter,
+                securityKeyButtonGetter,
                 continueButtonGetter
             ],
             app: app
         )
+
+        try waitForScreen()
```

Comment:
```suggestion
        try super.init(
            expectedElementGetters: [
                securityKeyButtonGetter,
                twoFAFieldGetter,
                continueButtonGetter
            ],
            app: app
        )
```

Instead of adding the `try waitForScreen` in `init`, I would suggest switching the order of `twoFAFieldGetter` and  `securityKeyButtonGetter` if the intention is for the existence of `securityKeyButton` to be checked on every test. This shouldn't make the test any less robust as the other two elements will be interacted with as part of the test (i.e., if they are missing, the test should still fail)

The reasoning for this suggestion is that `init` already does a `waitForScreen` but only for the first element in the `expectedElementGetters` array. Adding a call to `try waitForScreen()` in `init` adds about ~3 seconds to each test case in this test class. 

WDYT?

Reply:
Tanks @jostnes, I've updated this bit per your suggestion and added a comment to make it more clear!